app/models/config.rb

```diff
@@ -27,27 +16,28 @@ class Config
     hide_practical_support: 'Enter "yes" to hide the Practical Support panel on patient pages. This will not remove any existing data.'
   }.freeze
 
-  # Fields
-  enumerize :config_key, in: CONFIG_FIELDS
-  field :config_value, type: Hash, default: { options: [] }
-
-  # Indices
-  index({ config_key: 1 }, unique: true)
+  enum config_key: {
+    insurance: 0,
+    external_pledge_source: 1,
+    pledge_limit_help_text: 2,
+    language: 3,
+    resources_url: 4,
+    practical_support_guidance_url: 5,
+    fax_service: 6,
+    referred_by: 7,
+    practical_support: 8,
+    hide_practical_support: 9,
+    start_of_week: 10,
+    budget_bar_max: 11,
+    voicemail: 12,
+  }
 
   # Validations
   validates :config_key, uniqueness: true, presence: true
 
-  # History and auditing
-  track_history on: fields.keys + [:updated_by_id],
-                version_field: :version,
-                track_create: true,
-                track_update: true,
-                track_destroy: true
-  mongoid_userstamp user_model: 'User'
-
   # Methods
   def options
-    config_value[:options]
+    config_value['options']
```

Comment:
Why did this end up having to change?

Reply:
pg kicks them out stringified rather than symbolized.